share/p2p/shrexsub/pubsub.go

```diff
@@ -17,7 +17,7 @@ var log = logging.Logger("shrex-sub")
 
 // pubSubTopic hardcodes the name of the EDS floodsub topic with the provided suffix.
 func pubSubTopic(suffix string) string {
-	return "eds-sub/v0.0.1/" + suffix
+	return fmt.Sprintf("/eds-sub/v0.0.1/%s", suffix)
```

Comment:
are you sure this won't have a double `//` prepended?

Reply:
no. it should not